Finish "Resources" Section (#101)
Slimefun/Wiki: pages/Bucket-of-Fuel.md

> Buckets of Fuel are [Resource](https://github.com/Slimefun/Slimefun4/wiki/Resources)s that can be used to fuel certain [Machines](https://github.com/Slimefun/Slimefun4/wiki/Electric-Machines).

Reviewer: ```suggestion
A Bucket of Fuel is a [Resource](https://github.com/Slimefun/Slimefun4/wiki/Resources) that can be used to fuel certain [Machines](https://github.com/Slimefun/Slimefun4/wiki/Electric-Machines).
```
